src/yang/ietf-bfd-types.yang

```diff
@@ -84,19 +84,28 @@ module ietf-bfd-types {
   feature authentication {
     description
       "This feature indicates that the server supports BFD
-       authentication as defined in RFC 5880.";
+       authentication as defined in section 6.7 of RFC 5880.";
```

Comment:
Isn't this a repeat of the reference statement. We should in describe what we need to describe, and move all specific references to the reference section. In this case, it means deleting all the lines that contain reference information which is already present in the reference.

Reply:
So you're saying I should keep the description statement as it was before, the added reference statement (per Juergen's suggestion) is good enough?

Reply:
The modified description statement should read "This feature indicates that the server supports BFD authentication."